streams/br.m3u

```diff
@@ -24,6 +24,10 @@ https://stream01.msolutionbrasil.com.br/hls/canal25/live.m3u8
 https://arkyvbre1g.zoeweb.tv/fiocruz/fiocruz/playlist.m3u8
 #EXTINF:-1 tvg-id="CartoonNetworkBrazil.br" status="online",Cartoon Network (Spanish)
 https://playout.cdn.cartoonnetwork.com.br/playout_02/playlist.m3u8
+#EXTINF:-1 tvg-id="CartoonNetworkBrazil.br" status="online",Cartoon Network (Portuguese)
+https://playout.cdn.cartoonnetwork.com.br/playout_01/playlist.m3u8
+#EXTINF:-1 tvg-id="CartoonitoBrazil.br" status="online",Cartoonito (Portuguese)
+https://playout.cdn.cartoonnetwork.com.br/playout_03/playlist.m3u8
```

Comment:
please add this 
https://playout.cdn.cartoonnetwork.com.br/playout_04/playlist.m3u8

Do you know what the language is?

Reply:
Looks like Cartoonito in Spanish. Obviously the Spanish versions aren't actually Brazilian, but since they originate from Brazil and we're not sure exactly which country I guess it can stay there for now.

Reply:
added thanks